buildSrc/src/main/java/org/elasticsearch/gradle/plugin/PluginBuildPlugin.java

```diff
@@ -193,13 +141,8 @@ private static void configurePublishing(Project project, PluginPropertiesExtensi
 
     private static void configureDependencies(final Project project) {
         var dependencies = project.getDependencies();
-        if (BuildParams.isInternal()) {
-            dependencies.add("compileOnly", dependencies.project(Map.of("path", ":server")));
-            dependencies.add("testImplementation", dependencies.project(Map.of("path", ":test:framework")));
-        } else {
-            dependencies.add("compileOnly", "org.elasticsearch:elasticsearch:" + VersionProperties.getElasticsearch());
-            dependencies.add("testImplementation", "org.elasticsearch.test:framework:" + VersionProperties.getElasticsearch());
-        }
+        dependencies.add("compileOnly", "org.elasticsearch:elasticsearch:" + VersionProperties.getElasticsearch());
```

Comment:
We handle this differently and substitute it in our internal build to avoid using internal `BuildParams`